src/settings/target/mod.rs

```diff
@@ -120,6 +120,113 @@ impl Manifest {
         Ok(manifest)
     }
 
+    pub fn generate(
+        name: String,
+        target_type: TargetType,
+        config_path: PathBuf,
+    ) -> Result<Manifest, failure::Error> {
+        let mut manifest = Manifest::default();
+
+        manifest.name = name.clone();
+        manifest.target_type = target_type.clone();
+
+        manifest.route = Some(String::new());
+        manifest.workers_dev = Some(true);
+        manifest.zone_id = Some(String::new());
+
+        let toml = toml::to_string(&manifest)?;
+        let config_file = config_path.join("wrangler.toml");
+
+        log::info!("Writing a wrangler.toml file at {}", config_file.display());
+        fs::write(&config_file, &toml)?;
+        Ok(manifest)
+    }
+
+    pub fn generate_site(name: String, config_path: PathBuf) -> Result<Manifest, failure::Error> {
+        let site = Site::default();
+        let mut manifest = Manifest::default();
+
+        manifest.name = name.clone();
+        manifest.site = Some(site);
+
+        manifest.route = Some(String::new());
+        manifest.workers_dev = Some(true);
+        manifest.zone_id = Some(String::new());
+
+        let toml = toml::to_string(&manifest)?;
+        let config_file = config_path.join("wrangler.toml");
+
+        log::info!("Writing a wrangler.toml file at {}", config_file.display());
+        fs::write(&config_file, &toml)?;
+        Ok(manifest)
+    }
+
+    pub fn get_target(
```

Comment:
it looks like i did things to these methods. I did not, there is a refactor commit that simply moves the public function declarations to the top of the Manifest impl